Composer/packages/client/src/components/Notifications/NotificationCard.tsx

```diff
@@ -196,7 +259,10 @@ export const NotificationCard = React.memo((props: NotificationProps) => {
         ariaLabel={formatMessage('Close')}
         css={cancelButton}
         iconProps={{ iconName: 'Cancel', styles: { root: { fontSize: '12px' } } }}
-        onClick={() => onDismiss(id)}
+        onClick={() => {
+          cardProps?.onDismiss?.(id);
```

Comment:
This is what lets us track dismissals using the close box (or any other custom action we want to perform when the user closes a notification, relayed via card props).

Reply:
You should leave that as a comment in the code